README.md

```diff
@@ -14,6 +14,57 @@ Swift Tracing.
 
 As a stretch goal for this implementation we aim to pitch it to become embraced by the Jaeger project as an [officially supported library](https://www.jaegertracing.io/docs/1.19/client-libraries/#supported-libraries).
 
+## Installation
+
+First, Add the following package dependency to your `Package.swift` file:
+
+```swift
+.package(url: "https://github.com/slashmo/jaeger-client-swift.git", .branch("main")),
+```
+
+Then, add the `Jaeger` library to the target(s) you want to use it with:
+
+```swift
+.product(name: "Jaeger", package: "jaeger-client-swift"),
+```
+
+> 🔖 At this point in time no version has been tagged as the API is still very fragile. Make sure to watch the repository to stay tuned.
+
+## Configuration
+
+Now that you installed the Jaeger client you can use it in your project by bootstrapping the `InstrumentationSystem` with an instance of `JaegerClient`:
+
+```swift
+let recordingStrategy = // ... see instructions below
+let jaegerSettings = JaegerTracer.Settings(
+    serviceName: "frontend", 
+    recordingStrategy: recordingStrategy
+)
+let jaegerTracer = JaegerTracer(
+    settings: jaegerSettings, 
+    group: eventLoopGroup
+)
+InstrumentationSystem.bootstrap(jaegerTracer)
+```
+
+### Recording Strategy
```

Comment:
is this recording or _reporting_ (i.e. "exporting")? recording is inside the process, reporting/exporting is emitting to backends

Reply:
Correct, see comment above: https://github.com/slashmo/jaeger-client-swift/pull/1#discussion_r502337662